src/__tests__/sessionBackendService.test.ts

```diff
@@ -0,0 +1,184 @@
+import { OutsideExecution } from "@/outsideExecution"
+import { HttpResponse, http } from "msw"
+import { setupServer } from "msw/node"
+import { constants, stark } from "starknet"
+import { StarknetChainId } from "starknet-types"
+import { afterAll, afterEach, beforeAll, describe, expect, it } from "vitest"
+import { ArgentBackendSessionService } from "../sessionBackendService"
+import { BackendSignatureResponse } from "../sessionTypes"
+import { getSessionTypedData } from "../utils"
+
+export const restHandlers = [
+  http.post("https://api.hydrogen.argent47.net/v1/cosigner/signSession", () => {
```

Comment:
I would extract the base url to an env variable or at least a constant

Reply:
will do to a constant, it's used only for unit testing

will add the constant in the service too